coral-trino/src/main/java/com/linkedin/coral/trino/trino2rel/TrinoToRelConverter.java

```diff
@@ -94,12 +94,15 @@ protected RelNode standardizeRel(RelNode relNode) {
     return relNode;
   }
 
-  private static String trimParenthesis(String value) {
+  /*
+   * Standardizing Sql to ensure compatability with Sql Parser.
+   */
+  private static String standardizeSql(String value) {
     String str = value.trim();
     if (str.startsWith("(") && str.endsWith(")")) {
-      return trimParenthesis(str.substring(1, str.length() - 1));
+      return standardizeSql(str.substring(1, str.length() - 1));
     }
-    return str;
+    return str.toUpperCase().replaceAll("`", "\"");
```

Comment:
I believe putting replaceAll here should avoid unnecessary calls if we need to recurse, would appreciate confirmation on that.

Reply:
Why did we add `toUppercase()`. Also back quotes can be part of the literals so might accidentally change a backquote that was not supposed to be changed.

Reply:
The `toUpperCase()` is brought over from [this line](https://github.com/linkedin/coral/blob/346055432259d79180dae037f110a0c08c0e8464/coral-trino/src/main/java/com/linkedin/coral/trino/trino2rel/TrinoToRelConverter.java#L86).